### ISSUE TITLE
no Bulk_UnityEngine.UI_2.cpp file with older unity version (2017.2.2)

### ISSUE THREAD

**Issue by alejandrosklar** [yasirkula/UnityIonicIntegration#31]
What versions of Unity and Vuforia are supported? I'm asking because I tested this out initially with a Unity 2018.1.5 and Vuforia 7.1.35 and everything worked (I used xCode 8.3 to do most of the project configurations, and then opened the configured project in xCode 9.4 so that I could build for iOS 11.4, and everything worked beautifully). 

However, I realized the Unity project I need to use in my app does not work on newer unity versions, so I rebuilt it using Unity 2017.2.2 and Vuforia 6.5.25. Now I'm running into the following error:

`clang: error: no such file or directory $UNITY_BUILD_FOLDER/Classes/Native/Bulk_UnityEngine.UI_2.cpp`

and I see that the file is not included in the build directory from older Unity version. I've been trying to see if I can use the libil2cpp folder from the 2018.1.5 version build (or at least to copy over some of the include files), but I'm sort of going in circles because then other files are missing or renamed. 

I will continue trying to patch this together, and if no luck, I will attempt a fresh build from the very beginning. 

In the meantime, would be helpful to know:

-     Are Unity 2017.2.2 builds are even supported?

-     If they are supported, what could trying to include the "Bulk_UnityEngine.UI_2.cpp" file?

BTW, this repo is AWESOME, and its obvious you put a lot of work into it and are very much on top of responding to issues. Thanks so much for that. It's really amazing of you.

**Comment by yasirkula** [comment]
Thank you! To be honest, I've tested this plugin with Unity 5.6 only. I didn't need to use it on a newer project; so I'm unsure about compatibility with other versions.

I'm trying to understand why that Bulk_UnityEngine.UI_2.cpp error pops up. Does the error message contain any other useful information? Does it give the error while compiling one of the files that I provide? If you navigate to the Unity build folder, can you spot Bulk_UnityEngine.UI_2.cpp anywhere in there?

**Comment by alejandrosklar** [comment]
Ok, it turns out this is not an actual issue. Upon starting completely new xcode project, it worked. Bulk_UnityEngine.UI_2 file is not part of builds with Unity2017.2.2, and there must have been something leftover in xcode that was requiring trying to import it.

As for versions of Unity supported by this repo, I can verify that Unity projects (even ones with Vuforia) will work correctly for the following versions:
         - Unity2017.2.2
         - Unity2018.1.5

I did find that if the Unity project contains the Bluetooth LE for iOS, tvOS and Android package ([https://assetstore.unity.com/packages/tools/network/bluetooth-le-for-ios-tvos-and-android-26661](url)), it will not build in xCode. I was able to remove the package and get the project to build, so this is not a pending issue, but I will open a new issue in this repo, for the sake of completeness and to document.

Thanks again.
